docs/content/guides/technical-specification/mobile-support.md

```diff
@@ -0,0 +1,113 @@
+| [Row headers](@/guides/rows/row-header.md)                                 |     &check;      |   &check;    |   &check;    |
+| [Row parent-child](@/guides/rows/row-parent-child.md)                      |     &check;      |   &check;    |   &check;    |
+| [Row hiding](@/guides/rows/row-hiding.md)                                  |     &check;      |   &check;    |   &check;    |
+| [Row moving](@/guides/rows/row-moving.md)                                  |     &cross;      |   &cross;    |   &cross;    |
+| [Row freezing](@/guides/rows/row-freezing.md)                              |     &cross;      |   &cross;    |   &cross;    |
+| [Manual row resizing](@/guides/rows/row-height.md)                         |     &cross;      |   &cross;    |   &cross;    |
+| [Row virtualization](@/guides/rows/row-virtualization.md)                  |     &check;      |   &check;    |   &check;    |
+| [Rows sorting](@/guides/rows/row-sorting.md)                               |     &check;      |   &check;    |   &check;    |
+| [Row trimming](@/guides/rows/row-trimming.md)                              |     &check;      |   &check;    |   &check;    |
+| [Row pre-populating](@/guides/rows/row-prepopulating.md)                   |     &cross;      |   &cross;    |   &cross;    |
+| [Clipboard](@/guides/cell-features/clipboard.md)                           |     &check;      |   &check;    |   &check;    |
+| [Merge cells](@/guides/cell-features/merge-cells.md)                       |     &check;      |   &check;    |   &check;    |
+| [Conditional formatting](@/guides/cell-features/conditional-formatting.md) |     &check;      |   &check;    |   &check;    |
+| [Text alignment](@/guides/cell-features/text-alignment.md)                 |     &check;      |   &check;    |   &check;    |
+| [Disabled cells](@/guides/cell-features/disabled-cells.md)                 |     &check;      |   &check;    |   &check;    |
+| [Comments](@/guides/cell-features/comments.md)                             |     &cross;      |   &cross;    |   &cross;    |
+| [Autofill values](@/guides/cell-features/autofill-values.md)               |     &cross;      |   &cross;    |   &cross;    |
+| [Formula calculation](@/guides/formulas/formula-calculation.md)            |     &check;      |   &check;    |   &check;    |
+| [Context menu](@/guides/accessories-and-menus/context-menu.md)<sup>*</sup> |     &check;      |   &cross;    |   &cross;    |
+| [Undo and redo](@/guides/accessories-and-menus/undo-redo.md)               |     &check;      |   &check;    |   &check;    |
+| [Searching values](@/guides/accessories-and-menus/searching-values.md)     |     &check;      |   &check;    |   &check;    |
+| [Export to CSV](@/guides/accessories-and-menus/export-to-csv.md)           |     &cross;      |   &cross;    |   &cross;    |
+| [Layout direction](@/guides/internationalization/layout-direction.md)      |     &cross;      |   &cross;    |   &cross;    |
```

Comment:
It can be marked as "checked"  also, I forgot to mention it. I test scrolling with enabled 'rtl' direction on mobile.